plugin.go

```diff
@@ -187,20 +209,15 @@ func meta(environment, tags, keyringPassphrase string, username string, password
 	wg.Add(1)
 	go func(wg *sync.WaitGroup) {
 		defer wg.Done()
-		deployCmd := exec.Command("plasmactl", "platform:deploy", environment, tags)
+		deployCmd := exec.Command("plasmactl", "platform:deploy", environment, tags, verbosity)
```

Comment:
Interestingly,
`./plasmactl meta dev itneraction.bidule -vvv` works, but
`./plasmactl meta dev itneraction.bidule`  fails on step:
```
plasmactl platform:deploy dev itneraction.bidule
plasmactl platform:deploy environment tags [flags]
Error: accepts 2 arg(s), received 3
```
Not clear at all why action behaves that way. Possibly something about action definition makes launchrctl to pass `verbosity` empty string as an argument.

It does not matter much anyways as, for this action, we need to use `--debug` instead of `-vvv`

So line 212 should not use `verbosity` var but something else tailord to deploy action